tools/src/gt_mpi_gather.cc

```diff
@@ -431,6 +434,8 @@ void print_usage() {
             << "\t\t Optional, if workspace is specified in any of the json config files\n"
             << "\t \e[1m--array\e[0m=<array dir>, \e[1m-A\e[0m <GenomicsDB array dir>\n"
             << "\t\t Optional, if array is specified in any of the json config files\n"
+            << "\t \e[1m--transcriptomics-query\e[0m\n"
```

Comment:
Not sure, we want to overload `gt_mpi_gather`... Do we want a separate tool for querying transcriptomic data? I am fine, if this can be tackled when we complete the genomicsdb CLI later too.